src/psyclone/psyir/nodes/__init__.py

```diff
@@ -63,6 +63,8 @@
 # this package e.g. 'from psyclone.psyir.nodes import Literal'
 __all__ = [
         'Node',
+        'colored',
```

Comment:
Not the fault of this PR (and probably not to be addressed here either) but looking at this now I don't think 'colored' should be under psyir.nodes.

Reply:
I agree ... with everything. It shouldn't be there, and it's not to be addressed here :) Note that I only added this since in the past it worked because of a certain order of import statements (local imports, which I have moved to the top).

Reply:
I've created #913 for this.